tests/pebble.rs

```diff
@@ -130,7 +130,9 @@ struct Environment {
 
 impl Environment {
     /// Create a new [`Environment`] with running Pebble and challenge test servers.
-    async fn new(config: EnvironmentConfig) -> io::Result<Environment> {
+    async fn new() -> io::Result<Environment> {
+        let config = EnvironmentConfig::default();
```

Comment:
This change means it's not possible to customize the `EnvironmentConfig` for a test env above/beyond the default. That seems unnecessarily inflexible, no? It's not much savings to remove the one arg IMO.

Reply:
What about `Option<EnvironmentConfig>` with an `unwrap_or_default()` as a compromise?

Reply:
Dropped the commit. Not entirely sure when/why you'd want to customize the `EnvironmentConfig`...

Reply:
> Not entirely sure when/why you'd want to customize the EnvironmentConfig...

I was hoping to add a test soon that tests EAB account creation since it's a fairly tricky part of the spec with its own cryptography. That will require a pebble config with `externalAccountBindingRequired: true`. We can't set that in the default, because then we would have to use EAB for every test.

It's a bit "speculative" in that none of the existing tests need it, but I will promise to add a test for it soon :-)